### ISSUE TITLE
Never Pay Your Lawyer by the Hour

### ISSUE THREAD

**Issue by JattMannu** [JattMannu/react-blog-github#29]
## Incentive Super-Response Tendency

To control a rat infestation, French colonial rulers in Hanoi in the nineteenth century passed a law: For every dead rat handed in to the authorities, the catcher would receive a reward. Yes, many rats were destroyed, but many were also bred specially for this purpose.

In 1947, when the Dead Sea Scrolls were discovered, archaeologists set a finder’s fee for each new parchment. Instead of lots of extra scrolls being found, they were simply torn apart to increase the reward. Similarly, in China in the nineteenth century, an incentive was offered for finding dinosaur bones. Farmers located a few on their land, broke them into pieces, and cashed in. Modern incentives are no better: Company boards promise bonuses for achieved targets. And what happens? Managers invest more energy in trying to lower the targets than in growing the business.

These are examples of the incentive super-response tendency. Credited to Charlie Munger, this titanic name describes a rather trivial observation: People respond to incentives by doing what is in their best interests. What is noteworthy is, first, how quickly and radically people’s behavior changes when incentives come into play or are altered, and second, the fact that people respond to the incentives themselves, and not the grander intentions behind them.

Good incentive systems comprise both intent and reward. An example: In ancient Rome, engineers were made to stand underneath the construction at their bridges’ opening ceremonies. Poor incentive systems, on the other hand, overlook and sometimes even pervert the underlying aim. For example, censoring a book makes its contents more famous, and rewarding bank employees for each loan sold leads to a miserable credit portfolio. Making CEO pay public didn’t dampen the astronomical salaries; to the contrary, it pushed them upward. Nobody wants to be the loser CEO in his industry.

Do you want to influence the behavior of people or organizations? You could always preach about values and visions or you could appeal to reason. But in nearly every case, incentives work better. These need not be monetary; anything is possible, from good grades to Nobel Prizes to special treatment in the afterlife.

For a long time I tried to understand what made well-educated nobles from the Middle Ages bid adieu to their comfortable lives, swing themselves up onto horses, and take part in the Crusades. They were well aware that the arduous ride to Jerusalem lasted at least six months and passed directly through enemy territory; yet they took the risk. And then it came to me: The answer lies in incentive systems. If they came back alive, they could keep the spoils of war and live out their days as rich men. If they died, they automatically passed on to the afterlife as martyrs—with all the benefits that came with it. It was win-win.

Imagine for a moment that, instead of demanding enemies’ riches, warriors and soldiers charged by the hour. We would effectively be incentivizing them to take as long as possible, right? So why do we do just this with lawyers, architects, consultants, accountants, and driving instructors? My advice: Forget hourly rates and always negotiate a fixed price in advance.

Be wary, too, of investment advisers endorsing particular financial products. They are not interested in your financial well-being, but in earning a commission on these products. The same goes for entrepreneurs’ and investment bankers’ business plans. These are often worthless because, again, the vendors have their own interests at heart. What is the old adage? “Never ask a barber if you need a haircut.”

In conclusion: Keep an eye out for the incentive super-response tendency. If a person’s or an organization’s behavior confounds you, ask yourself what incentive might lie behind it. I guarantee you that you’ll be able to explain 90 percent of the cases this way. What makes up the remaining 10 percent? Passion, idiocy, psychosis, or malice.
